### ISSUE TITLE
Newer Hugo versions

### ISSUE THREAD

**Issue by Travolter** [vividvilla/ezhil#50]
I'm running Hugo v0.68 and the theme isn't rendering properly. Specifically the recent posts aren't showing up as you can see here: https://hurrdurrimasheep.be/p/ezhil.png

it used to work for v0.57 but on a different machine running v0.68 the same setup does not work anymore. Is this a known issue? I noticed the sample website is using v0.51 so I figured maybe it's not being tested on newer hugo versions?

**Comment by komish** [comment]
@Travolter Can you share what this used to look like on v0.57 for comparison?  Been doing some testing and my recent posts seem to render as I would expect.

**Comment by Travolter** [comment]
Well, the latest blogposts should show up: https://hurrdurrimasheep.be/p/ezhil_old.png

**Comment by devidasjadhav** [comment]
@Travolter I am facing similar issue. 
change/ update your theme directory to latest theme directory. 
it solved my problem. 
I would like to know what resolved it?
So I can only patch that to my blog.
find screen shot attached. 
![Screenshot from 2020-08-18 09-37-15](https://user-images.githubusercontent.com/7537047/90469503-9b682a00-e136-11ea-8797-122491160b32.png)
![Screenshot from 2020-08-18 09-37-58](https://user-images.githubusercontent.com/7537047/90469506-9c995700-e136-11ea-8454-e9c86946f4d3.png)

my hugo version 
Hugo Static Site Generator v0.68.3-157669A0

**Comment by devidasjadhav** [comment]
I have changed `.Data.Pages` to `.Site.RegularPages`.

```diff
diff --git a/themes/ezhil/layouts/index.html b/themes/ezhil/layouts/index.html
index fefffe4..d5d32b5 100644
--- a/themes/ezhil/layouts/index.html
+++ b/themes/ezhil/layouts/index.html
@@ -11,7 +11,7 @@
                        </h2>
                        <div class="posts">
                                {{- $.Scratch.Set "counter" 0 -}}
-                               {{- range .Data.Pages -}}
+                               {{- range .Site.RegularPages -}}
                                {{- if (in (.Site.Params.excludedTypes | default (slice "page")) .Type) -}}
                                {{- else -}}
                                {{- if lt ($.Scratch.Get "counter") (.Site.Params.RecentPostsCount | default 10) -}}
```

and resolved the problem.

Newer version of ezhil theme has pagination now.
So this issue is not relevant and I can't raise PR.
feel free to use this patch.
If that works close this issue.

**Comment by vividvilla** [comment]
Closing this since it was fixed here bb28e76551834a70f1889fb7601b49e5abd7e328
